From b3f24c96640db9d01c073c5d61600a88e52b5dd5 Mon Sep 17 00:00:00 2001
From: xoviat <xoviat@users.noreply.github.com>
Date: Thu, 11 Jan 2018 11:34:53 -0600
Subject: [PATCH] CMake: Use the correct library name on windows

FindBLAS and FindLAPACK don't correctly detect the OpenBLAS library
name on windows. I am unsure why this was originally set this way, but
it has caused me some trouble.
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1bdcd52ee..b5789119a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,11 +12,7 @@ set(OpenBLAS_VERSION "${OpenBLAS_MAJOR_VERSION}.${OpenBLAS_MINOR_VERSION}.${Open
 # Adhere to GNU filesystem layout conventions
 include(GNUInstallDirs)
 
-if(MSVC)
-set(OpenBLAS_LIBNAME libopenblas)
-else()
 set(OpenBLAS_LIBNAME openblas)
-endif()
 
 #######
 if(MSVC)
